docs/index.md

```diff
@@ -0,0 +1,12 @@
+# PowerShell-Docker
+
+Hello!
+This is the main documentation page for PowerShell-Docker, so here you can find some helpful details for different questions.
+
+## Image Purpose
+
+These images are built so that PowerShell users can run the program in a containerized environment - see [this article](https://opensource.com/resources/what-docker) for what Docker is, and some basic pros and cons.
```

Comment:
1. The main image is intended to be minimal.  In other words, if you bring a module, you may need to add libraries to the image.

2. The test-dep sub images are intended to allow running tests in Azure Dev Ops